packages/core/src/provider/base.ts

```diff
@@ -175,15 +181,37 @@ export class SmartAccountProvider<
         );
       }
 
-      // TODO: need to add support for overriding gas prices
       return {
         target: request.to,
         data: request.data ?? "0x",
         value: request.value ? fromHex(request.value, "bigint") : 0n,
       };
     });
 
-    const { hash } = await this.sendUserOperation(batch);
+    const bigIntMax = (...args: bigint[]) => {
+      if (!args.length) {
+        return undefined;
+      }
+
+      return args.reduce((m, c) => (m > c ? m : c));
```

Comment:
I believe you should have an "initialValue"? [(docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce). Perhaps 0?

Reply:
in the case of initialValue not being supplied it uses `args[0]` as the first value and starts with `c` as `args[1]`